cli/test.js

```diff
@@ -23,49 +22,19 @@ function test(command, argv) {
   const specsPath = path.resolve(process.cwd(), 'src/app/**/*.spec.ts');
   const specsGlob = glob.sync(specsPath);
 
-  let lintResult;
-
-  const onRunStart = () => {
-    localeAssetsProcessor.prepareLocaleFiles();
-    lintResult = tsLinter.lintSync();
-  };
-
-  const onRunComplete = () => {
-    if (lintResult && lintResult.exitCode > 0) {
-      // Pull the logger out of the execution stream to let it print
-      // after karma's coverage reporter.
-      setTimeout(() => {
-        logger.error('Process failed due to linting errors:');
-        lintResult.errors.forEach(error => logger.error(error));
-      }, 10);
-    }
-  };
-
   const onExit = (exitCode) => {
-    if (exitCode === 0) {
-      if (lintResult) {
-        exitCode = lintResult.exitCode;
-      }
-    }
-
     logger.info(`Karma has exited with ${exitCode}.`);
     process.exit(exitCode);
   };
 
-  const onBrowserError = () => {
```

Comment:
Email SPA especially didn't like this callback. Omnibar SPA had no problem with it.

Reply:
Is that because that SPA had a browser error?  I wonder if it's the fact that we're hooking in to this event, or what we're doing inside the handler for it?